core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.html

```diff
@@ -46,7 +46,7 @@
     </div>
     <p-scrollPanel [style]="{ width: '100%', height: '10.9375rem' }">
         <div class="template-builder-box__item" *ngFor="let item of items; let i = index">
-            <p>{{ item.identifier }}</p>
+            <p>{{ containerMap[item.identifier]?.title }}</p>
```

Comment:
```suggestion
            <p>{{ containerMap[item.identifier]?.title || item.identifier }}</p>
```

Just in case... we can show something.

Reply:
Also this needs testing.